### DISCUSSION TITLE
7E3 Model

### DISCUSSION THREAD

**Discussion by dzianismr** [Open-Systems-Pharmacology/Forum#1415] [Models] [answered]
Hi @Christoph27,

Thanks for sharing models from your PK-Sim BMOL publication (Niederalt et al 2018), really nice work. However the model shared for 7E3 antibody (https://github.com/Christoph27/7E3-Model/) seems to be SMAL molecule rather than BMOL model.

Best Regards, Denis

**Comment by dzianismr** [comment]
It looks like all BMOL repositories except for Tefibazumab contain the same SMAL model. I assume these are placeholders, will wait for the actual release.

**Reply by Christoph27** [reply]
Hi Denis, the models are meanwhile available on GitHub.
Best, Christoph

**Reply by dzianismr** [reply]
Thanks Christoph

**Comment by Christoph27** [comment, marked as answer]
I'm still working on the models for GitHub. Should be finished very soon. At the moment they are indeed placeholders. When finished, they will be available as evaluation plans at OSP repository. 

Best,
Christoph
